PeerioTechnologies/peerio-desktop
# fix: folder navigation bugs

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/665/desktop-ux-design-folder-navigation
https://app.clubhouse.io/peerio/story/690/desktop-ux-design-move-files

#### Testing instructions  
Pretty straightforward:
1. In Files view, check that folder/files "move" button does not show up if there are no folders.
2. In MoveFileDialog, check that folders that have no children do not show the `>` button to navigate into that folder.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
